specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,392 @@
         }
       }
     },
+    "potentialOutagesGetResult": {
+      "description": "The Get PotentialOutages operation response.",
+      "type": "object",
+      "$ref": "#/definitions/potentialoutage"
+    },
+    "potentialoutage": {
+      "type": "object",
+      "description": "Potential outages detected by Brain.",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Id for the potential outage.",
+          "format": "int64"
+        },
+        "title": {
+          "type": "string",
+          "description": "Title for the potential outage."
+        },
+        "summary": {
+          "type": "string",
+          "description": "Summary for the potential outage."
+        },
+        "confidence": {
+          "type": "number",
+          "description": "Confidence score for the potential outage."
+        },
+        "displayConfidence": {
+          "type": "string",
+          "description": "Display confidence score for the potential outage."
+        },
+        "plane": {
+          "type": "string",
+          "description": "Type of the potential outage: control plane or data plane."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the potential outage."
+        },
+        "correlatedIncidentCount": {
+          "type": "integer",
+          "description": "Number of correlated incidents in the potential outage.",
+          "format": "int64"
+        },
+        "firstReportTime": {
+          "type": "string",
+          "description": "Time and date the potential outage started to become a potential outage."
+        },
+        "lastReportTime": {
+          "type": "string",
+          "description": "Time and date the potential outage was created."
+        },
+        "rawIds": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "Ids of child raw potential outages."
+        },
+        "duplicateIds": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "Duplicate consolidated outage id."
+        },
+        "outageDeclared": {
+          "type": "boolean",
+          "description": "A value indicating whether there is outage declared for this potential outage."
+        },
+        "expired": {
+          "type": "boolean",
+          "description": "A value indicating whether the potential outage is expired."
+        },
+        "incidents": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/icmIncidents"
+          },
+          "description": "A list of outage incidents."
+        },
+        "isSynthetic": {
+          "type": "boolean",
+          "description": "a value indicating whether the entity is real.No value means non-synthetic."
```

Comment:
Is this a nullable field? Shouldn't be better to always set a value to either true or false instead of mentioning that no value means non-synthetic (false)?

Reply:
i think it's a better way to handle it. addressed, thanks!

Reply:
What does it mean that an entity is "real"?

---
In reply to: [551342424](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r551342424) [](ancestors = 551342424)

Reply:
Can you clarify the description?

---
In reply to: [552322458](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r552322458) [](ancestors = 552322458,551342424)